.github/ISSUE_TEMPLATE/4-hosting-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: ":cloud: Hosting request"
+labels: hosting-request
+about: I want to host my repo in keptn-sandbox
+---
+
+### Description
+
+<!-- Some context around your integration here -->
+
+### Checklist
+- [ ] Repository transfer: <your-repo-here>
+- [ ] GitHub access permissions
+- [ ] Repository author has followed the [hosting guide](https://github.com/keptn-sandbox/contributing/blob/master/CONTRIBUTING.md) 
```

Comment:
Replaced `Integration author` with `Repository author` because hosting requests are not limited to Integrations (e.g., GSoC projects)